### ISSUE TITLE
Cannot Set Learning Rate and Optimizer but Have to Tune

### ISSUE THREAD

**Issue by hwds2023** [keras-team/autokeras#1866]
Hi,
I'm using autokeras.AutoModel to tune the model structure but found that I cannot set the optimizer and learning rate, which have to been tuned automatically. Actually I just want set them rather than tune. Is there any way or sample code to show how to set optimizer and learning rate?
Many thanks!
